hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/OMClientRequest.java

```diff
@@ -265,4 +272,24 @@ public AuditMessage buildAuditMessage(AuditAction op,
     auditMap.put(OzoneConsts.VOLUME, volume);
     return auditMap;
   }
+
+  @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
+  public static String getNormalizedKey(boolean enableFileSystemPaths,
```

Comment:
What about moving the 'enableFileSystemPaths' flag outside, and call the method conditionally? In that way, the getNormalizedKey will stay a simple util function without knowing about application logic.

Reply:
That requires in all over the code, we need to check this flag and call this. I feel this is better way.
If needed I can move the code of actual Normalize, to another utility if you expect this to be used else where.

Reply:
Moved common logic to a new method.